test/e2e-tests.sh

```diff
@@ -63,6 +63,12 @@ function set_feature_gate() {
   jsonpatch=$(printf "{\"data\": {\"enable-api-fields\": \"%s\"}}" $1)
   echo "feature-flags ConfigMap patch: ${jsonpatch}"
   kubectl patch configmap feature-flags -n tekton-pipelines -p "$jsonpatch"
+  if [ "$gate" == "alpha" ]; then
+    echo "enable feature flags in alpha phase"
+    jsonpatch=$(printf "{\"data\": {\"enable-cel-in-whenexpression\": \"true\"}}")
+    echo "feature-flags ConfigMap patch: ${jsonpatch}"
+    kubectl patch configmap feature-flags -n tekton-pipelines -p "$jsonpatch"
+  fi
```

Comment:
I wonder if this is the best practice or if its better to set it in https://github.com/tektoncd/pipeline/blob/main/test/e2e-tests-kind-prow-alpha.env

Reply:
That is a good idea, I think we can combine them in 1 func

Reply:
Updated, ptal